docs/001_develop/03_client-capabilities/010_filters/foundation-filters.mdx

```diff
@@ -0,0 +1,437 @@
+---
+title: 'Foundation Filters'
+sidebar_label: 'Foundation Filters'
+id: foundation-filters
+keywords: [genesis, foundation, ui, filters]
+tags:
+  - filter
+  - feature
+  - flags
+  - conditions
+---
+
+import FiltersDemo from '../../../../examples/ui/client-capabilities/filters/filters.js';
+
+# Genesis Foundation Filters
+
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lerna.js.org/)
+[![TypeScript](https://img.shields.io/badge/%3C%2F%3E-TypeScript-%230074c1.svg)](https://www.typescriptlang.org/)
+
+`foundation-filters` provides a collection of client-side filters.
+
+Client-side filters in Genesis allow users to filter and process data locally within the web application. They may also access runtime data or state to provide an outcome. These filters, including options like userTargeting and timeWindow, enable customised data manipulation directly in the browser, enhancing responsiveness by reducing server requests.
+
+Client-side filters generally take primitive input parameters, and return `true` or `false` based on these values.  They can be combined using the ClientFilterRunner<!-- TODO:(./docs/api/foundation-filters.runner.md)--> utility provided in the `foundation-filters` package to create layered filtering conditions for complex data scenarios.When a filter is used without all the input parameters needed, Genesis logs a warning and returns `true` by convention so that the chain can continue. This is currently controlled at an individual filter level, and may change in time.
+
+:::important
+It's important to highlight that filters are purposefully decoupled from any `effect`, like pairing with a `when()` directive or custom-effect logic. This flexibility allows these filters to be reused for numerous purposes.
```

Comment:
I don't think this block is necessary for the docs, I think this is more technical info that should just be annotated on the cde